### PR TITLE
12.0 mig partner default journal

### PR DESCRIPTION
**Pull request by VrajaTechnologies**

### CODE REVIEW

**Comment by eantones** [comment]
@VrajaTechnologies Please, squash the "Migration to 12.0" commits into one, and force push it again. It should only be 5 commits in total. Thanks

**Comment by eantones** [comment]
Hi @VrajaTechnologies Travis is ok here, you just need to squash the 2 commits with the name "Migration 12.0", without it, we cannot see clearly the changes made. If you need help, we could make a call tomorrow 29th at your 14:30 (our 11:00). Let me know

File: partner_default_journal/views/res_partner_views.xml [hunk 1]
```diff
@@ -8,12 +8,10 @@
         <field name="model">res.partner</field>
         <field name="inherit_id" ref="account.view_partner_property_form" />
         <field name="arch" type="xml">
-            <group name="acc_sale" position="inside">
```
Comment (eantones) [inline partner_default_journal/views/res_partner_views.xml]: @VrajaTechnologies Please, just replace "acc_sale" for "sale" and leave the rest unchanged

File: partner_default_journal/views/res_partner_views.xml [hunk 2]
```diff
@@ -8,12 +8,10 @@
...
+            <xpath expr="//field[@name='property_account_payable_id']" position="after">
                 <field name="sale_journal_id" options="{'no_create_edit': True}" />
-            </group>
-            <group name="acc_purchase" position="inside">
```
Comment (eantones) [inline partner_default_journal/views/res_partner_views.xml]: @VrajaTechnologies, please, just replace "acc_purchase" for "purchase" and leave the rest unchanged

File: partner_default_journal/models/account_invoice.py [hunk 3]
```diff
@@ -12,39 +12,36 @@ class AccountInvoice(models.Model):
...
+        elif self.partner_id.purchase_journal_id and self.type in [
             "in_invoice",
             "in_refund",
-        ):
+        ]:
             self.journal_id = self.partner_id.purchase_journal_id
 
         if (
             not self.partner_id.sale_journal_id
-            and not self.partner_id.purchase_journal_id
```
Comment (eantones) [inline partner_default_journal/models/account_invoice.py]: @VrajaTechnologies Why did you changed that???

File: partner_default_journal/models/res_partner.py [hunk 4]
```diff
@@ -2,15 +2,17 @@
...
-from odoo import api, fields, models
+from odoo import fields, models
 
 
 class ResPartner(models.Model):
     _inherit = "res.partner"
 
     sale_journal_id = fields.Many2one(
-        "account.journal", "Default journal", domain=[("type", "=", "sale")]
+        "account.journal", "Default Sale journal", domain=[("type", "=", "sale")]
```
Comment (eantones) [inline partner_default_journal/models/res_partner.py:12]: With the changes made on the view I already commented, you don't need to rename the this field, please leave it as it was. The changes should be always minimal.

File: partner_default_journal/models/res_partner.py [hunk 5]
```diff
@@ -2,15 +2,17 @@
...
     )
     purchase_journal_id = fields.Many2one(
-        "account.journal", "Default journal", domain=[("type", "=", "purchase")]
+        "account.journal",
+        "Default Purchase journal",
```
Comment (eantones) [inline partner_default_journal/models/res_partner.py:16]: Same as sale journal

**Comment by eantones** [comment]
Hi @VrajaTechnologies when this PR was done and merged, I'm gonna send you more modules to migrate. Thanks

**Review by eantones — CHANGES_REQUESTED** [review]

**Comment by eantones** [comment]
@VrajaTechnologies please, squash "Migration 12.0" commits to a single one using rebase -i as we discuss in one of the calls. Thanks

**Comment by VrajaTechnologies** [comment]
Hello
Rebase successfully done..please check it and let me know If anything is
missing.
Thanks


Best Regards,

Vraja Technologies

Vraja Technologies
*Odoo ERP Consultancy Service Provider*
207, Time Square, Ayodhya Chowk, • 150 feet ring road, Rajkot - 360006
*India*.
*Skype* :  *live:vrajatechnologies*
[image: email-icon]  ***@***.***
***@***.***>*
[image: website-icon]  *www.vrajatechnologies.com
<https://www.vrajatechnologies.com/>*

**Comment by eantones** [comment]
Hi @VrajaTechnologies I still see two commits
image

 it should only be one "Migration 12.0" commit. Please squash them using rebase -i and force a push
Thanks

**Comment by VrajaTechnologies** [comment]
Hi Eric,

We tried so many times to push forcefully. Also Tried to rebase the
committee but not getting successful. I got an error when I pushed
forcefully.

Can you please create a meeting with us so we will stay on call and fix it
quickly and start the next module.

It would be really helpful if you could connect with us at the meeting.

Thanks In Advance


Best Regards,

Vraja Technologies

Vraja Technologies
*Odoo ERP Consultancy Service Provider*
207, Time Square, Ayodhya Chowk, • 150 feet ring road, Rajkot - 360006
*India*.
*Skype* :  *live:vrajatechnologies*
[image: email-icon]  ***@***.***
***@***.***>*
[image: website-icon]  *www.vrajatechnologies.com
<https://www.vrajatechnologies.com/>*

**Comment by eantones** [comment]
Hi @VrajaTechnologies , can you send me a screenshot with the error please. Thanks

**Comment by VrajaTechnologies** [comment]
Hi Nuobit/Odoo-Addons,

We tried to check the rebase module but error faced. Please find an
attachment .

Also if you have steps to resolve this issue then please let us know.

Thanks In Advance



[image: Screenshot from 2021-06-30 16-22-58.png]


[image: Screenshot from 2021-06-30 16-23-09.png]


Best Regards,

Vraja Technologies

Vraja Technologies
*Odoo ERP Consultancy Service Provider*
207, Time Square, Ayodhya Chowk, • 150 feet ring road, Rajkot - 360006
*India*.
*Skype* :  *live:vrajatechnologies*
[image: email-icon]  ***@***.***
***@***.***>*
[image: website-icon]  *www.vrajatechnologies.com
<https://www.vrajatechnologies.com/>*

**Comment by eantones** [comment]
@VrajaTechnologies the screenshots are not in the message, can you please send them again. Thanks

**Comment by VrajaTechnologies** [comment]
Best Regards,

Vraja Technologies

Vraja Technologies
*Odoo ERP Consultancy Service Provider*
207, Time Square, Ayodhya Chowk, • 150 feet ring road, Rajkot - 360006
*India*.
*Skype* :  *live:vrajatechnologies*
[image: email-icon]  ***@***.***
***@***.***>*
[image: website-icon]  *www.vrajatechnologies.com
<https://www.vrajatechnologies.com/>*

**Comment by eantones** [comment]
@VrajaTechnologies still nothing, can you please send them by e-mail. Thanks

**Comment by VrajaTechnologies** [comment]
Please find an attachment. I have attached a zip file. you need to extract
it.


Best Regards,

Vraja Technologies

Vraja Technologies
*Odoo ERP Consultancy Service Provider*
207, Time Square, Ayodhya Chowk, • 150 feet ring road, Rajkot - 360006
*India*.
*Skype* :  *live:vrajatechnologies*
[image: email-icon]  ***@***.***
***@***.***>*
[image: website-icon]  *www.vrajatechnologies.com
<https://www.vrajatechnologies.com/>*

**Comment by eantones** [comment]
@VrajaTechnologies I'm not receiving any attach, please send them by e-mail directly. Thanks

**Comment by eantones** [comment]
Hi @VrajaTechnologies now there's 3 additional commits instead of 1. I've already done it, I'm closing this PR please check the solution on new PR https://github.com/nuobit/odoo-addons/pull/11. Thanks

**Pull request closed without merging**